ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -365,14 +363,37 @@ private static String getHeader(Map<String, Object> args, String header, boolean
    * Looks at the URI to determine the type of operation required or the blob ID that an operation needs to be
    * performed on.
    * @param restRequest {@link RestRequest} containing metadata about the request.
-   * @return extracted operation type or blob ID from the uri.
+   * @param subResourcePresent the {@link RestUtils.SubResource} if one is present. {@code null} otherwise.
```

Comment:
We could name the variable "subResource". Its understandable it will be null if not sub-resource is present